assets/texts/fr/permissions.md

```diff
@@ -5,21 +5,20 @@ https://support.mozilla.org/fr/kb/messages-de-demande-de-autorisation-pour-les-e
 
 ## Autorisations d'installation
 
-Actuellement, aucune autoristation n'est requise à l'installation ou lors des mises à jour.
+Actuellement, aucune autorisation n'est requise à l'installation ou lors des mises à jour.
 
 ## Autorisations spécifiques à certaines fonctionnalités (optionnelles)
 
-Ces autoristations sont requises pour effectuer certaines actions, si elles sont nécéssaires.
+Ces autorisations sont requises pour effectuer certaines actions, si elles sont nécessaires.
 
-|  ID Interne | Autorisations                                                                           | Demandé au …                              | Explication  |
-|:------------|:----------------------------------------------------------------------------------------|:------------------------------------------|:-------------|
-| `downloads` | Télécharge les fichiers, et lit et modifie l'historique de téléchargement du navigateur | Téléchargement de QR code en tant que SVG | Needed for … |
+|    ID Interne    | Autorisations                           | Demandé au/à …                                                                    | Explication                                                                                                                                                                 |
+|:-----------------|:----------------------------------------|:----------------------------------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `clipboardWrite` | Écrit des données vers le presse-papier | L'activation de l'option de ne copier les émojis uniquement si l'insertion échoue | Utilisée pour copier les émojis de manière asynchrone, _seulement_ si l'insertion échoue (si vous voulez tout le temps les copier, il n'y a pas besoin de cette permission).|
```

Comment:
"L'activation de l'option de ne copier les émojis uniquement si l'insertion échoue" 
I think some word(s) are missing, or some should be erased.

Reply:
I tried to adjust it, but I would like your opinion on it; I'm not sure it sounds clear enough